### PR TITLE
Fixed volume profiling issue

### PR DESCRIPTION
**Pull request by gnehapk**
tendrl-github-id: https://github.com/Tendrl/ui/issues/959

Signed-off-by: Neha Gupta <gnehapk@gmail.com>

### CODE REVIEW

**Review by r0h4n — CHANGES_REQUESTED** [review]
@gnehapk Every commit msg requires a bugzilla ID, please dont merge PRs without that

File: src/commons/stores/volume-store.js [hunk 1]
```diff
@@ -61,8 +61,10 @@
                         profileStatus = "Pending";
                     } else if (data[i].profiling_enabled === "yes") {
```
Comment (cloudbehl) [inline src/commons/stores/volume-store.js]: rather then if-else can we write a JSON and do it?
something like:
{"no": "Disabled",
"yes": "Enabled",
"unknown": "Unknown"}

File: src/commons/stores/cluster-store.js [hunk 2]
```diff
@@ -306,6 +306,10 @@
                 status = "Pending";
             } else {
                 status = profileStatus[cluster.volume_profiling_state];
+
+                if(typeof status === "undefined") {
+                    status = "Unknown";
```
Comment (cloudbehl) [inline src/commons/stores/cluster-store.js:311]: Status coming in lower case so Unknown should be lower case too

**Review by cloudbehl — CHANGES_REQUESTED** [review]

File: src/commons/stores/volume-store.js (hunk 1 above)
Comment (gnehapk) [reply src/commons/stores/volume-store.js]: @cloudbehl We are sure of "yes" and "no" value only. Apart from that 3rd value can be blank/null/none. Hence, need to do this way.

File: src/commons/stores/cluster-store.js (hunk 2 above)
Comment (gnehapk) [reply src/commons/stores/cluster-store.js:311]: @cloudbehl its used for showing purpose in UI.

**Review by cloudbehl — DISMISSED** [review]

**Review by a2batic — APPROVED** [review]

**Pull request merged**